### PR TITLE
Add missing mouse wheel masks to the ButtonList enum

### PR DESCRIPTION
**Pull request by dewolen**
The global enum ButtonList was missing four mask values.
Bits 1 - 3, 8 - 9 were defined, while 4 - 7 were not.

Added:
```
BUTTON_MASK_WHEEL_UP = 8
BUTTON_MASK_WHEEL_DOWN = 16
BUTTON_MASK_WHEEL_LEFT = 32
BUTTON_MASK_WHEEL_RIGHT = 64
```

### CODE REVIEW

File: core/input/input_event.h [hunk 1]
```diff
@@ -55,6 +55,10 @@ enum ButtonList {
 	BUTTON_MASK_LEFT = (1 << (BUTTON_LEFT - 1)),
 	BUTTON_MASK_RIGHT = (1 << (BUTTON_RIGHT - 1)),
 	BUTTON_MASK_MIDDLE = (1 << (BUTTON_MIDDLE - 1)),
+	BUTTON_MASK_UP = (1 << (BUTTON_WHEEL_UP - 1)),
+	BUTTON_MASK_DOWN = (1 << (BUTTON_WHEEL_DOWN - 1)),
```
Comment (dewolen) [inline core/input/input_event.h]: Typo: next commit changes those to BUTTON_MASK_WHEEL_UP and BUTTON_MASK_WHEEL_DOWN

**Comment by akien-mga** [comment]
Is there a use case for those masks beyond filling the blank? I guess if they were not added, it's probably that they're not needed/relevant.

**Comment by dewolen** [comment]
@akien-mga Well, they're as useful as any other mask. I guess they weren't added because scroll wheel "buttons" are instantaneous (last one frame) and can't be held down. You can always use `InputEventMouseButton.button_index` instead of `InputEventMouse.button_mask`, so they're not necessary. I've decided to add them after someone on Discord had trouble with finding them, but they also ended up using `button_index`. This was my first ever PR so I might have not thought about whether I should add them. If you don't consider including them maybe we could add a simple comment to the cpp file saying something like `// masks 8, 16, 32, 64 correspond to wheel up, down, left and right masks respectively, not included due to few use cases`

**Comment by aaronfranke** [comment]
@dewolen Is this still desired? If so, it needs to be [rebased](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) and tested on the latest master branch. I see from the discussion that it may not be desired, so if that's the case then you can just close this.

**Pull request closed without merging**